examples/hello_gcs/hello_gcs.cc

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <google/cloud/functions/internal/framework.h>
+#include <google/cloud/storage/client.h>
+#include <sstream>
+
+using ::google::cloud::functions::HttpRequest;
+using ::google::cloud::functions::HttpResponse;
+namespace gcs = ::google::cloud::storage;
+
+HttpResponse HelloGcs(HttpRequest request) {  // NOLINT
+  auto error = [] {
+    auto constexpr kError = 400;
+    HttpResponse response;
+    response.set_result(kError);
+    return response;
+  };
+
+  std::vector<std::string> components;
+  std::istringstream split(request.target());
+  for (std::string c; std::getline(split, c, '/'); components.push_back(c))
+    ;
```

Comment:
Opt: I might suggest `{}` or `continue;` there.

Reply:
Done, fixed the title too.